src/hotspot/share/gc/shenandoah/shenandoahNumberSeq.hpp

```diff
@@ -50,6 +50,9 @@ class HdrSeq: public NumberSeq {
 
   virtual void add(double val);
   double percentile(double level) const;
+
+  // Merge this HdrSeq into hdr2, optionally clearing this HdrSeq
+  void merge(HdrSeq& hdr2, bool clear_this = true);
```

Comment:
The default setting here is based on the way its only current client (RS scan instrumentation) makes use of it, but I am happy to change it if reviewers feel that might be better for API hygiene reasons.